lib/DDG/Spice/NobelPrize.pm

```diff
@@ -0,0 +1,43 @@
+package DDG::Spice::NobelPrize;
+
+# ABSTRACT: Write an abstract here
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+use Data::Dumper;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'http://api.nobelprize.org/v1/prize.json?' 
+                . '&year=$1' 
+                . '&category=$2';
+
+
+# Triggers - https://duck.co/duckduckhack/spice_triggers
+triggers startend => 'nobel';
+
+spice from => '([^-]+)-([^-]+)';
+
+# Handle statement
+handle query_clean => sub {
```

Comment:
I would refactor + simplify this. The regex are constant so we can define them outside our `handle`

I would do: 
```perl
package DDG::Spice::NobelPrize;

use DDG::Spice;
use strict;

spice is_cached => 1;
spice proxy_cache_valid => "200 1d";

spice wrap_jsonp_callback => 1;

spice from => "([^-]+)?-([^-]+)?";
spice to => 'http://api.nobelprize.org/v1/prize.json?year=$1&category=$2';

triggers startend => 'nobel';

my $year_regex = qr/\b(?<year>\d{4})\b/;
my $category_regex = qr/\b(?<category>physics|chemistry|medicine|peace|literature|economics)\b/;

handle remainder_lc => sub {

	return unless $_;
    return unless m/$year_regex|$category_regex/;

    my $year = $+{year} || "";
    my $category = $+{category} || "";

    return "$year-$category";
};

1;
```